README.adoc

```diff
@@ -266,6 +266,28 @@ class UsersController < ApplicationController
 end
 ----
 
+=== Action Filters Order [[action-filters-order]]
+
+Order controller filter declarations in the order in which they will be executed.
```

Comment:
This is getting complicated with inheritance and concerns.
When it comes to a point where order matters, isn’t it a sign that a different approach is needed?

Reply:
I don't think that the action filters order will significantly impact the complexity of the code. Instead, it provides clarity in understanding the order in which the action filters are executed.

I try to avoid inheriting controllers other than inheriting from the application controller.
And as for the concerns, I haven't had any problems with them other than this [one](https://github.com/rubocop/rubocop-rails/issues/1047).